src/routes/(app)/settings/memory/+page.server.ts

```
@@ -0,0 +1,9 @@
+import type { PageServerLoad } from './$types';
+import { heapStats } from 'bun:jsc';
+
+export const load: PageServerLoad = async ({}) => {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)